packages/ui/src/components/Accordion/Accordion.tsx

```diff
@@ -48,6 +48,26 @@ const StyledHeader = React.forwardRef<
   />
 ));
 
+type AccordionHeaderPrimitivePropsWithoutRef = React.ComponentPropsWithoutRef<
+  typeof AccordionPrimitive.Header
+>;
+export interface AccordionHeaderProps
+  extends AccordionHeaderPrimitivePropsWithoutRef {
+  headingAs?: HeadingRank;
+}
+
+const Header = React.forwardRef<
+  React.ElementRef<typeof AccordionPrimitive.Header>,
+  AccordionHeaderProps
+>(({ headingAs = "h2", children, ...rest }, forwardedRef) => {
+  const Heading = headingAs;
+  return (
+    <StyledHeader asChild ref={forwardedRef} {...rest}>
+      <Heading data-h2-line-height="base(1)">{children}</Heading>
```

Comment:
We will need to pass `headingAs` to this underlying component. Right now everything shows up at `h2` regardless of what is passed in.

Reply:
Are you sure?  I popped an h3 into Storybook and it seemed to work OK.  Am I missing something?   :sweat_smile: 
![image](https://user-images.githubusercontent.com/8978655/234385853-e4950816-11c0-4130-bab9-fb85b29ff724.png)

Reply:
Wow, I'm sorry. I completely missed line 64 there and was looking at the wrong story! 🤦‍♀️ 😛